contracts/Forwarder.sol

```diff
@@ -81,6 +96,23 @@ contract Forwarder {
     );
   }
 
+  /**
+   * Execute a nft transfer from the forwarder to the parent address
+   * @param tokenContractAddress the address of the ERC721 NFT contract
+   * @param tokenId The token id of the nft
+   */
+  function flushERC721Tokens(address tokenContractAddress, uint256 tokenId) external onlyParent {
+    IERC721 instance = IERC721(tokenContractAddress);
+    address forwarderAddress = address(this);
+    uint256 ownerAddress = instance.ownerOf(tokenId);
+
+    if (forwarderAddress != ownerAddress) {
+      return;
+    }
+
+    instance.transferFrom(forwarderAddress, parentAddress, tokenId);
```

Comment:
using transferFrom since nft didn't use safeTransferedFrom to send nft since it needs flushed

Reply:
I think this is fine since we save some gas